pkg/config/config.go

```diff
@@ -22,6 +25,8 @@ type Config struct {
 	LogLevel     string
 	Profile      Profile
 	ProfilesFile string
+
+	location string
```

Comment:
How is `location` different from `ProfilesFile`? It appears that they store the same thing, can we just re-use one of them for all purposes?

Reply:
`ProfilesFile` currently only stores the profile location when it's user supplied, not the implicit location used by the CLI. I'd be worried about overriding that behavior.